src/test/java/org/testeditor/fixture/host/integrationtest/HostDriverIntegrationTest.java

```diff
@@ -121,4 +122,44 @@ public void typeAtTest() {
         hostDriverFixture.disconnect();
     }
 
+    @Test
+    public void newTransactionTest() {
+        assumeTrue();
```

Comment:
We will decide what to do with this test, so if we decide to delete it  :cry: , changes make no sense.